lib/scripts/controllers/options.js

```diff
@@ -56,8 +59,13 @@ export class OptionCtrl {
       console.log(data);
       this.patterns = JSON.parse(data.savedPatterns);
       this.key = data.yandexTranslatorApiKey;
+
       this.blacklistedWords = data.userBlacklistedWords.slice(1, -1).split('|');
+      this.blacklistedWords = this.blacklistedWords.filter(function(n){ return n != '' });
+
       this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
+      this.blacklistedWebsites = this.blacklistedWebsites.filter(function(n){ return n != '' });
```

Comment:
@rohankatyal29 why are these required? Can't we make sure that we don't blacklist blank words. That would be a better check. What do you think?